src/main/java/storagesystem/model/Location.java

```diff
@@ -2,6 +2,12 @@
 
 import javafx.scene.image.Image;
 
+/**
+ * Location tells us information about a place where a part of a team's inventory is.
```

Comment:
An instance doesn't necessarily have to be for a part of a team's inventory.

```suggestion
 * Location tells us information about a place.
```